Dockerfile

```diff
@@ -58,3 +58,10 @@ RUN SWIFT_URL=https://swift.org/builds/$SWIFT_BRANCH/$(echo "$SWIFT_PLATFORM" |
 
 # Print Installed Swift Version
 RUN swift --version
+
+WORKDIR /code
+ONBUILD COPY Package.swift /code/
+ONBUILD RUN swift build -c release
+ONBUILD COPY Sources /code/
+ONBUILD RUN swift build -c release
+CMD ["/bin/bash", "-c", "find .build -executable -type f -not -name '*.so' | head | bash"]
```

Comment:
This is super clever! However I've noticed that some dependencies include executables, so this might find those examples by accident.

Two ideas:
1. Sort by creation time. The last built executable binary is probably(?) the user's own binary.
2. `grep` `swift package describe` for `Type: executable`, since that will list the non-dependency executables.

`swift package describe` can also output in json, but we'd need `jq` installed for that...